### PR TITLE
Fix release zipfile

### PR DESCRIPTION
**Pull request by rustyrussell**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -230,7 +232,7 @@ There predate the BOLT specifications, and are only of vague historic interest:
 7. [0.5.2] - 2016-11-21: "Bitcoin Savings & Trust Daily Interest II"
 
 [Unreleased]: https://github.com/ElementsProject/lightning/compare/v0.6.2...HEAD
-[0.6.3-rc1]: https://github.com/ElementsProject/lightning/compare/v0.6.3-rc1...HEAD
+[0.6.3]: https://github.com/ElementsProject/lightning/compare/v0.6.3...HEAD
```
Comment (jb55) [inline CHANGELOG.md]: should this line be:  `https://github.com/ElementsProject/lightning/releases/tag/v0.6.3` ?

Comment (jb55) [reply CHANGELOG.md]: and `[Unreleased]: https://github.com/ElementsProject/lightning/compare/v0.6.3...HEAD`

Comment (rustyrussell) [reply CHANGELOG.md]: Yeah, I fuxked up that should not have been here at all.  Will force push

**Comment by jb55** [comment]
if you have a fresh zip file generated from this I would be interested in testing it. for now I'm still using recursive clone in https://github.com/NixOS/nixpkgs/pull/53732

**Comment by cdecker** [comment]
Small fixup to fix the `shellcheck` checks about unquoted values. `$FILES` could not be quoted since we actually want that var to be split, so I opted to silence the error instead.

**Comment by rustyrussell** [comment]
Rebased onto master and folded @cdecker fix in.

**Pull request merged**